dhcpv6/dhcpv6message.go

```diff
@@ -288,6 +288,18 @@ func (mo MessageOptions) DHCP4oDHCP6Server() *OptDHCP4oDHCP6Server {
 	return nil
 }
 
+// NTP returns the NTP Server option as defined by RFC 5908.
+func (mo MessageOptions) NTP() *OptNTPServer {
```

Comment:
There are three different types of sub-options in an NTP Server option, hence returning the whole thing. I can think of two alternatives:
1) leave it as it is: return the option, and let the user filter from `o.Suboptions`
2) Add more functions, like `NTPSrvAddr() []net.IP`, `NTPMCAddr() []net.IP`, and `NTPSrvFQDN() string`

What do you think?

Reply:
I am ok either way. I would go with option 2 and only implement the function for subtype 1, which is likely the one that going to be used the most. The user can get the rest anyway. However, I don't have a strong opinion.

Reply:
sounds good, I'll implement `NTPServers() []net.IP` which returns the sub-option 1 servers, and has a more descriptive name. Let me know if you prefer otherwise

Reply:
one more thing: the RFC states that a packet can have multiple NTP server options. I'll implement this function to iterate over all the NTP server options it finds, and concatenate the server addresses in order